x-pack/plugins/rule_registry/server/utils/rbac.ts

```diff
@@ -9,9 +9,14 @@
  * registering a new instance of the rule data client
  * in a new plugin will require updating the below data structure
  * to include the index name where the alerts as data will be written to.
+ *
+ * This doesn't work in combination with the `xpack.ruleRegistry.index`
+ * setting, with which the user can change the index prefix.
  */
 export const mapConsumerToIndexName = {
   apm: '.alerts-observability-apm',
```

Comment:
should this be changed to `.alerts-observability.apm`?

Reply:
Yes, but it's not what this PR is about. It needs to be changed in multiple places. It's tracked in #102089.